### PR TITLE
Add setup guide for Kubernetes deployment

### PR DESCRIPTION
**Pull request by zainasir**
Add installation guide for the official cbioportal helm charts.

# Checks
- [x] The commit log is comprehensible. It follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). We can fix this during merge by using a squash+merge if necessary
- [x] Has tests or has a separate issue that describes the types of test that should be created. If no test is included it should explicitly be mentioned in the PR why there is no test.
- [x] Is this PR adding logic based on one or more **clinical** attributes? If yes, please make sure validation for this attribute is also present in the data validation / data loading layers (in backend repo) and documented in [File-Formats Clinical data section](https://github.com/cBioPortal/cbioportal/blob/master/docs/File-Formats.md#clinical-data)!
- [x] Make sure your PR has one of the labels defined in https://github.com/cBioPortal/cbioportal/blob/master/.github/release-drafter.yml

### CODE REVIEW

File: docs/deployment/README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ Private instances of cBioPortal are maintained by institutions and companies [ar
 
 An instance can be deployed using Docker **(recommended)** or by building and deploying from source. The source code of cBioPortal is [available](https://github.com/cBioPortal/cbioportal) on GitHub under the terms of [Affero GPL V3](https://www.gnu.org/licenses/agpl-3.0.en.html).
 
-This section contains instructions for both of these paths.
+To deploy cBioPortal instance on a Kubernetes cluster, we also provide [official Helm charts](https://artifacthub.io/packages/search?org=cbioportal). You can follow the installation guide [here]().
```
Comment (inodb) [inline docs/deployment/README.md:7]:
```suggestion
To deploy cBioPortal instance on a Kubernetes cluster, we also provide [official Helm charts](https://artifacthub.io/packages/search?org=cbioportal). You can follow the installation guide [here](./kubernetes/README.md).
```

File: docs/deployment/retype.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+input: .
```
Comment (inodb) [inline docs/deployment/retype.yml:1]: do we need these additional `retype.yml` files?

File: docs/retype.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+input: .
```
Comment (inodb) [inline docs/retype.yml:1]: do we need these additional `retype.yml` files?

**Review by inodb — COMMENTED** [review]
Looks good! Made a few small comments

File: docs/retype.yml (hunk 3 above)
Comment (zainasir) [reply docs/retype.yml:1]: These were auto-generated by retype but they are not needed unless we want to do additional customization. I'll delete them.

**Pull request merged**